src/main/java/lotto/domain/model/Rank.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain.model;
+
+public enum Rank {
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
+    MISS(0, 0);
+
+    private int countOfMatch;
+    private int winningMoney;
+
+    private Rank(int countOfMatch, int winningMoney) {
+        this.countOfMatch = countOfMatch;
+        this.winningMoney = winningMoney;
+    }
+
+    public static Rank valueOf(int countOfMatch, boolean matchBonus) {
+        if (countOfMatch == 6) return FIRST;
```

Comment:
```suggestion
        if (countOfMatch == FIRST.countOfMatch) return FIRST;
```
`6`이라는 값을 직접 사용하는 대신 FIRST enum 객체의 값을 사용해 보면 어떨까요?